src/structs/Api.ts

```diff
@@ -3,6 +3,8 @@ import ApiError from '../utils/ApiError'
 import qs from 'querystring'
 import { EventEmitter } from 'events'
 
+import { Snowflake, BotStats, BotInfo, UserInfo, BotsResponse, ShortUser, BotsQuery } from '../typings'
+
```

Comment:
Are these unexported types still accessible from `./dist/typing.d.ts`?

Reply:
All of the types are exported in typings.ts

Reply:
So yes you can do
```ts
import {  BotStats } from '@top-gg/sdk/dist/typings'
```

Reply:
oh I see what the issue was then ok